app/models/asset.rb

```diff
@@ -77,12 +79,32 @@ def unique_id
     object_id
   end
 
+  # MySQL encoding is set to utf8, which only supports Basic Multilingual Plane characters.
+  UNSUPPORTED_UTF_8 = %r{[^\u0000-\uffff]}.freeze
```

Comment:
Thanks CodeClimate, Rubocop says the exact opposite:

```
app/models/asset.rb:83:23: C: Style/MutableConstant: Freeze mutable objects assigned to constants.
  UNSUPPORTED_UTF_8 = %r{[^\u0000-\uffff]}
                      ^^^^^^^^^^^^^^^^^^^^
```

Reply:
Heh, code climate is just rubocop here, must be version/settings difference.

Reply:
I can't update CodeClimate to Ruby 2.6 because they don't support it yet. Not sure what I can do to fix it. Do we want the error on CodeClimate or locally?

Reply:
I removed the code, which also solves the problem.